pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -42,6 +42,17 @@ rec {
     openSha256 = "sha256-etbtw6LMRUcFoZC9EDDRrTDekV8JFRYmkp3idLaMk5g=";
     settingsSha256 = "sha256-8KB6T9f+gWl8Ni+uOyrJKiiH5mNx9eyfCcW/RjPTQQA=";
     persistencedSha256 = "sha256-zrstlt/0YVGnsPGUuBbR9ULutywi2wNDVxh7OhJM7tM=";
+
+    prePatch = "pushd kernel";
+    postPatch = "popd";
```

Comment:
```suggestion
    patchFlags = [ "-p1" "-d" "kernel" ];
```

```
       -d dir  or  --directory=dir
          Change to the directory dir immediately, before doing anything else.
```

Reply:
iirc I used `prePatch`/`postPatch` in a previous PR to avoid causing a mass-rebuild of all the nvidia drivers, but since we're about to hit a release it makes sense to make the change now. Will add a commit once I've finished running `nixpkgs-review`